packages/tiptap/extensions.ts

```diff
@@ -15,7 +18,14 @@ export const tipTapExtensions: AnyExtension[] = [
   }),
   Link.configure({
     linkOnPaste: true,
-    autolink: true
+    autolink: true,
+    openOnClick: true,
+    HTMLAttributes: {
+      class: 'href-link'
```

Comment:
you can just use tailwind classnames here, and remove it from global css

Reply:
This happens at runtime and not build time

Reply:
still should be fine, we have been doing it like always,
If it creates any issues in staging we can always revert it.

Reply:
also don't use `text-blue-400`, we have disabled those colors
use `text-blue-${1..12}` from radix color pallete